index.html

```diff
@@ -6983,7 +6983,7 @@ <h1>Diamond Sponsors</h1>
                     <div class="col-md-4 col-sm-6 sponsor-column">
                         <div class="sponsor text-center">
                             <a href="https://www.google.com/cloud/" target="_self">
-                                <img alt="Google" src="img/GoogleCloud.png">
+                                <img alt="Google" src="img/google-cloud-logo.png">
```

Comment:
@abhishek71994 use ```class="partner-logo"```, this particular style is made for making logos comparable in size.

Reply:
@simsausaurabh adding partner-logo as class is not working.

Reply:
@abhishek71994 You will have to remove sponsor class from the div just above containing img tag. Then, it should work.